src/app-layout/__tests__/multi-layout.test.tsx

```diff
@@ -84,10 +84,11 @@ describeEachAppLayout({ themes: ['refresh-toolbar'], sizes: ['desktop'] }, () =>
         content={<AppLayout data-testid="second" navigationHide={true} tools="testing tools" />}
       />
     );
-    expect(firstLayout.findTools()).toBeFalsy();
-    expect(secondLayout.findTools()).toBeFalsy();
+    expect(isDrawerClosed(firstLayout.findTools())).toEqual(true);
```

Comment:
This code was meant to assert there is no tools panel on that instance at all, because `toolsHide={true}`. Not sure if `isDrawerClosed` is correct here

Reply:
Ok, I think there is something going on with this test suite or the current implementation. For a sanity check, I switched to the main branch and added `expect(firstLayout.findTools()).toBeFalsy();` after `expect(secondLayout.findTools()).toBeTruthy();`, after the toggle is clicked, and it fails returning a panel with the same panel content as the `secondLayout`. I've noticed a couple flakey assertions like that in this file, which makes me think there are some other issues at play.